db/migrate/20150903010750_add_research_librarians_to_system_configuration.rb

```diff
@@ -0,0 +1,5 @@
+class AddResearchLibrariansToSystemConfiguration < ActiveRecord::Migration
+  def change
+    add_column :system_configurations, :research_librarians, :string, default: ""
```

Comment:
You don't need a migration for each field you add... You can put both fields in one migration.